web/src/Sidebar.tsx

```diff
@@ -9,6 +9,7 @@ class SidebarItem {
   name: string
   status: string
   hasWarnings: boolean
+  hasEndpoints: boolean
```

Comment:
is this supposed to be in a different branch?

Reply:
nope! now that the tiltfile is at the top, it's not very helpful to open a preview of the tiltfile by default